content/installation/image-builder.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: bt_wiki
+title: Using the Image Builder
+category: Installation
+draft: false
+weight: 600
+
+---
+
+{{% gsNote title="Supported Clouds" %}}
+ * AWS
+ * OpenStack
+ * vSphere
+{{% /gsNote %}}
+
+{{% gsNote title="Prerequisites" %}}
+ * [packer](https://www.packer.io/intro/getting-started/setup.html) += 0.8.6 (older versions not tested)
```

Comment:
Might be worth to put this as 'tested on <versions>' rather than "We didn't test on anything older than this"